assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-footer-block/block.tsx

```diff
@@ -50,6 +56,8 @@ const Block = ( {
 		  parseInt( cartTotals.total_items_tax, 10 )
 		: parseInt( cartTotals.total_items, 10 );
 
+	const cartItemUpdating = isCartItemUpdating();
```

Comment:
I believe you should use useSelect() here, not just select(). Also in the cart/proceed-to-checkout-block the method which is called is isCalculating() from the checkout store. I would keep it consistent. There could be other types of calculations (e.g. deleting a cart item) which are not handled by getItemsPendingQuantityUpdate()

Reply:
You're right, updated, thanks!